### PR TITLE
Split doc build into two

### PR DESCRIPTION
**Pull request by steven1046**
#### What was fixed?  (Issue # or description of fix)
This is needed in the short term because the docs build is taking over 60m. It will help with pipelining many builds queued up as well.
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] Dymanic Accessability Plugin (DAP)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

**Review by mbroz2 — CHANGES_REQUESTED** [review]
I would recommend keeping docs.sh until you deliver the two docs scripts; otherwise you'll break all the pipelines until you get them all updated. 
Once everything is delivered and updated, then open another PR to remove docs.sh

**Comment by steven1046** [comment]
@mbroz2 i fixed it.

File: scripts/build/docs_part_1.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+set -e
+export BUILD_SCRIPTS_DIR=$(dirname $0)
+echo "BUILD_SCRIPTS_DIR: $BUILD_SCRIPTS_DIR"
+
+echo "Begin building all doc content"
```
Comment (mbroz2) [inline scripts/build/docs_part_1.sh]: I'd remove this echo

File: scripts/build/docs_part_2.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+set -e
+export BUILD_SCRIPTS_DIR=$(dirname $0)
+
+timer_start=$(date +%s)
+python3 $BUILD_SCRIPTS_DIR/ToC_parse_features.py
+timer_end=$(date +%s)
+echo "Total execution time for parsing ToC Features: '$(date -u --date @$(( $timer_end - $timer_start )) +%H:%M:%S)'"
+
+echo "Finished building and prepping all Antora content"
```
Comment (mbroz2) [inline scripts/build/docs_part_2.sh]: I would move this echo to part_1 or remove/change it

**Review by mbroz2 — DISMISSED** [review]

**Pull request merged**